index.html

```diff
@@ -139,6 +139,15 @@ <h4>Restrictable tracks</h4>
               <var>T</var> is either a {{MediaStreamTrack}} returned inside of a {{MediaStream}}
               through a call to {{MediaDevices/getDisplayMedia()}}, or a clone of such a track.
             </li>
+            <li>
+              <var>T</var> is associated with a
+              <a data-cite="screen-capture/#dfn-browser">browser</a>
+              <a data-cite="screen-capture/#dfn-display-surface">dispaly surface</a>. (That is, if
+              <var>T</var>.{{MediaStreamTrack/getSettings()}} were called, it would have returned a
```

Comment:
Maybe something "simpler" as `<var>T</var>.{{MediaStreamTrack/getSettings()}}.{{MediaTrackSettings /displaySurface}} == {{DisplayCaptureSurfaceType/"browser"}}`

Reply:
My concern is that it implies that `getSettings()` has to be invoked, and:
1. That fits better in an algorithm than in a definition.
2. Methods invoked may have side effects.

Reply:
Got it.